### PR TITLE
Adds varied forms of bones and fat to zombie-creatures and humanoids.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
~No more boneless zombies and people!~

New items, accessed through either butchering or crafting:

**tainted bones** - Half as efficient for crafting as normal/human bones, except for charcoal. Generated at half the usual rate for most monsters. Can't be used in edible/foodsafe recipes.
**tainted fat** - Must be made into tainted tallow to be useful. Less zombie-creatures have the FAT tag, so this is less common than normal fat.
**tainted tallow** - Craftable. Requires twice as much tainted fat to produce as normal tallow. Can't be used in edible/foodsafe recipes.
**human bones** - Can't be used in edible recipes (could theoretically use it in some sort of cannibal aspic or something, though). Has all the usual ups and downs associated with cannibalism.

All new butchering outputs use existing monster flags.

### CODE REVIEW

File: data/json/monsters.json [hunk 1]
```diff
@@ -2491,7 +2491,7 @@
     "death_function":"MELT",
     "special_attacks":[["LONGSWIPE", 8]],
     "description":"Intense radiation has spurred a unique form of necrosis and regeneration, it is impossible to tell if this creature was ever human.",
-    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "VIS50", "NOGIB"]
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "VIS50", "NOGIB", "BONES"]
```
Comment (KA101) [inline data/json/monsters.json]: Uh, the Necropolis folks melt on death, so whether they have bones is academic.  Just something to keep in mind.

File: data/json/monsters.json [hunk 2]
```diff
@@ -5619,7 +5619,7 @@
     "death_function":"JACKSON",
     "special_attacks":[["JACKSON", 0]],
     "description":"And though you fight to stay alive, \nYour body starts to shiver. \nFor no mere mortal can resist, \nThe evil of the thriller.",
-    "flags":["SEES", "SMELLS", "WARM", "BASHES", "BLEED", "NO_BREATHE", "VIS50"]
+    "flags":["SEES", "SMELLS", "WARM", "BASHES", "BLEED", "NO_BREATHE", "VIS50", "BONES", "POISON"]
```
Comment (KA101) [inline data/json/monsters.json]: Jackson's not a hulk.  Thinking FAT is misplaced.

File: data/json/monsters.json (hunk 1 above)
Comment (Rivet-the-Zombie) [inline data/json/monsters.json]: Yeah that's just there so if the player were to figure out some means of preventing it, they'd still have bones like they ought to.

File: data/json/monsters.json (hunk 2 above)
Comment (Rivet-the-Zombie) [inline data/json/monsters.json]: The MJ zombie doesn't have the FAT tag. Backup dancers do, since they're hulks.

Comment (KA101) [inline data/json/monsters.json]: OK, must've misread.

**Comment by Snaaty** [comment]
So, I guess you left sinews out on purpose, because the zombie bodies are too decayed in order to salvage anything that could be used for example for sewing?

**Comment by KA101** [comment]
That's what I'd hold.

**Comment by Rivet-the-Zombie** [comment]
Yes.

**Pull request merged**